packages/perseus/src/widgets/label-image/label-image-validator.ts

```diff
@@ -43,14 +43,15 @@ export function scoreMarker(
     return score;
 }
 
+// TODO(LEMS-2440): Can possibly remove rubric parameter during 2440?
```

Comment:
For LabelImage I don't think we'll be able to remove rubric. It'll be something like:

```TS
type PerseusLabelImageUserInput = {
    selected: ReadonlyArray<string>;
}

type PerseusLabelImageRubric = {
    answers: ReadonlyArray<string>;
}
```

We just have to untangle how LabelImage is mashing together WidgetOptions/Rubric/UserInput. No action needed right now, it'll be a part of LEMS-2440.

Reply:
Updated the comment to point out where `answers` is located.